### PR TITLE
Fix some initial setup requirements

### PR DESCRIPTION
**Pull request by remi-dupre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,10 +78,8 @@ We have to fetch them or generate them using scripts. To do that, run:
         MIDDLEWARE += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
    
 
-* By default a SQLite3 (file based) database is used. Disclaimer: this is
-  convenient but *performance is poor*. It is thus recommended to setup a local
-  Postgres database and use that instead. If you do choose to use Postgres,
-  change `dev.py` accordingly:
+ * It is required to setup a local Postgres database and use that instead.
```
Comment (seirl) [inline README.md]: Why "instead"?

Shouldn't this be earlier in the README?

File: README.md [hunk 2]
```diff
@@ -55,6 +56,21 @@ Then, check the following settings in the file you copied:
...
 
+   
+It is required to setup a local Postgres database.
+Modify database's connection accordingly:
+
+     :::python
+     DATABASES = {
+         'default': {
+             'ENGINE': 'django.db.backends.postgresql_psycopg2',
+             'HOST': 'localhost',
```
Comment (seirl) [inline README.md]: Can we remove HOST, USER and PASSWORD? People should use peer auth instead, it's easier to configure.

File: README.md [hunk 3]
```diff
@@ -55,6 +56,17 @@ Then, check the following settings in the file you copied:
...
    empty for most tests.
 
+   
+It is required to setup a local Postgres database.
+Modify database's connection accordingly:
+
+     :::python
+     DATABASES = {
+         'default': {
+             'ENGINE': 'django.db.backends.postgresql_psycopg2',
```
Comment (zopieux) [inline README.md]: Just `postgresql`

File: README.md [hunk 4]
```diff
@@ -55,6 +56,17 @@ Then, check the following settings in the file you copied:
 1. Uncomment and update `RECAPTCHA_{PUBLIC,PRIVATE}_KEY`. You can leave them
    empty for most tests.
 
+   
+It is required to setup a local Postgres database.
+Modify database's connection accordingly:
```
Comment (zopieux) [inline README.md]: No `'s`

**Review by zopieux — APPROVED** [review]

File: prologin/prologin/settings/conf.sample.py [hunk 5]
```diff
@@ -27,6 +27,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
+        'HOST': 'localhost',
```
Comment (seirl) [inline prologin/prologin/settings/conf.sample.py]: Why?

**Review by seirl — CHANGES_REQUESTED** [review]

File: prologin/prologin/settings/conf.sample.py (hunk 5 above)
Comment (remi-dupre) [reply prologin/prologin/settings/conf.sample.py]: It wasn't consistant with the readme, I preferred verbosity

**Pull request merged**